src/app/app.module.ts

```diff
@@ -231,7 +231,9 @@ export function initAuthentication(moduleConfig: OAuthModuleConfig, oauthService
       redirectUri: environment.authentication.redirectUri ? environment.authentication.redirectUri : window.location.origin + '/',
```

Comment:
Now that we set the `redirectUri` in each environment configuration we should remove the fallback here. The URI is simply a mandatory value in the environment settings now.

Reply:
Thank you for the suggestion 👍 

I have set `redirectUri` to `window.location.href` directly in the app module and removed the fallback.